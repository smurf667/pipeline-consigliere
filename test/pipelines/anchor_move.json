{
  "error": 0,
  "warn": 2,
  "info": 0,
  "fixes": true
}
