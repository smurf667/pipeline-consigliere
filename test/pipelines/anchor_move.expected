.cache_key: &cache_key
  key: hello
.cache_path: &extra_paths
  - abc
  - xyz
default:
  interruptible: true
  cache:
    <<: *cache_key # include this
    paths:
      <<: *extra_paths
workflow:
  rules:
    - if: $CI_PIPELINE_SOURCE == "merge_request_event"
    - if: $CI_COMMIT_BRANCH && $CI_OPEN_MERGE_REQUESTS
      when: never
    - if: $CI_COMMIT_BRANCH
  auto_cancel:
    on_new_commit: interruptible
world:
  stage: build
  image: busybox
  timeout: 1 second
  rules:
    - if: $WEIRD == 'true'
